vars/buildDockerAndPublishImage.groovy

```diff
@@ -125,7 +115,7 @@ def call(String imageShortName, Map userConfig=[:]) {
       "IMAGE_DIR=${finalConfig.imageDir}",
       "IMAGE_DOCKERFILE=${finalConfig.dockerfile}",
       "BUILD_TARGETPLATFORM=${finalConfig.targetplatforms.split(',')[0]}",
-      "BAKE_TARGETPLATFORMS=${finalConfig.targetplatforms}",
```

Comment:
Only JSON require the last item not to have a ending comma.

When writing lists with 1 item per line, it is safer to keep the trailing comma so if any line inversion happens then no syntax error are trigger.